Airwallex/Core/Sources/Internal/Extensions/NSData+Base64.m

```diff
@@ -0,0 +1,25 @@
+//
+//  NSData+Base64.m
+//  Core
+//
+//  Created by Hector.Huang on 2023/3/17.
+//  Copyright © 2023 Airwallex. All rights reserved.
+//
+
+#import "NSData+Base64.h"
+
+@implementation NSData (Base64)
+
++ (instancetype)initWithBase64NoPaddingString:(NSString *)base64String {
```

Comment:
Can we have some individual tests for this extension?

Reply:
Just added.